Docs/material.md

```diff
@@ -288,24 +290,24 @@ void BlinnPhongMaterial::registerMaterial() {
         []( Ra::Engine::RenderTechnique& rt, bool isTransparent ) {
             // Configuration for RenderTechnique::LIGHTING_OPAQUE (Mandatory)
             Ra::Engine::ShaderConfiguration lpconfig(
-                "BlinnPhong", "Shaders/Materials/BlinnPhong/BlinnPhong.vert.glsl",
-                "Shaders/Materials/BlinnPhong/BlinnPhong.frag.glsl" );
+                "BlinnPhong", resourcesRootDir+"Shaders/Materials/BlinnPhong/BlinnPhong.vert.glsl",
+                resourcesRootDir+"Shaders/Materials/BlinnPhong/BlinnPhong.frag.glsl" );
             Ra::Engine::ShaderConfigurationFactory::addConfiguration( lpconfig );
             rt.setConfiguration( lpconfig, Ra::Engine::RenderTechnique::LIGHTING_OPAQUE );
 
             // Configuration for RenderTechnique::Z_PREPASS
             Ra::Engine::ShaderConfiguration dpconfig(
-                "DepthAmbiantBlinnPhong", "Shaders/Materials/BlinnPhong/BlinnPhong.vert.glsl",
-                "Shaders/Materials/BlinnPhong/DepthAmbientBlinnPhong.frag.glsl" );
+                "DepthAmbiantBlinnPhong", resourcesRootDir+"Shaders/Materials/BlinnPhong/BlinnPhong.vert.glsl",
+                resourcesRootDir+"Shaders/BlinnPhong/DepthAmbientBlinnPhong.frag.glsl" );
```

Comment:
should the path have `Materials` in it or not?

Reply:
?

Reply:
There is `Shaders/Materials/BlinnPhong/BlinnPhong.vert.glsl`, with `Materials`, and `Shaders/BlinnPhong/DepthAmbientBlinnPhong.frag.glsl` without. Which is the correct one?

Reply:
very old typo, corrected